template_feed/Microsoft.DotNet.Web.ProjectTemplates.2.0/content/WebApi-CSharp/Areas/IdentityService/Views/IdentityService/LoggedOut.cshtml

```diff
@@ -0,0 +1,5 @@
+@{
+    ViewData["Title"] = "Sign Out";
```

Comment:
Yeah this has always been a mess. It just looks weird in some places because in the same line/method you see both terms used. Not a ship-stopper.